### ISSUE TITLE
Question on Dimensions of Images

### ISSUE THREAD

**Issue by mtmarsh2** [nyanp/mnisten#2]
I know that usually Mnist images are loaded in as grayscale, so only one dimesion. Can this library be used for making files from 3 dimensional color images too? Thanks!

**Comment by nyanp** [comment]
@mtmarsh2 
Yes, but images are always converted to the single dimensional grayscale.
